strip_binaries.py

```diff
@@ -5,23 +5,50 @@
 import subprocess
 
 basepath = sys.argv[1]
+debugroot = sys.argv[2]
+if (debugroot[-1] == os.path.sep) and (len(debugroot) > 1):
+    debugroot = debugroot[:-1]
 
 files = {}
 
 paths = [basepath]
 
-# def read_retval(command):
-#     pr = subprocess.run(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-#     retval = pr.returncode
-#     if retval != 0:
-#         data = None
-#     else:
-#         data = pr.stdout.decode('utf8')
-#     return retval, data
+def read_retval(command):
+    # command must be a list of strings
+    pr = subprocess.run(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    retval = pr.returncode
+    if retval != 0:
+        data = None
+    else:
+        data = pr.stdout.decode('utf8')
+    return retval, data
+
+def read_elf_type(filepath):
+    retval, data = read_retval(['file', filepath])
+    if retval != 0:
+        return None, None
+    filetype = None
+    pos = data.find('BuildID')
+    if pos == -1:
+        BuildID = None
+    else:
+        BuildID = data[1+data.find('=',pos):data.find(',', pos)]
+    if ('LSB pie executable' in data) or ('LSB executable' in data):
```

Comment:
You might want to ensure `LC_ALL=C` is set, otherwise you're going to run into troubles (I did)

Reply:
Good point!

Reply:
Now I do everything in python, so this should not be a problem anymore.